04/index.html

```diff
@@ -1,30 +1,50 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/72220b4ed7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width:761px)" href="./styles/tablet.css">
+
 </head>
+
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
+
+    <nav class="nav">
+        <input class="burger__input" type="checkbox" id="menu-switcher">
+        <label class="burger__btn" for="menu-switcher"></label>
```

Comment:
Skoro mamy elementy tj. `burger__input`, `burger__btn` to powinien być gdzieś block `burger` - brakuje go.
Ponieważ trudno będzie go tutaj osadzić to ja bym zrobił element dla `nav` tj. `nav__burger-input` oraz `nav__burger-btn`